apps/api/src/app/auth/services/passport/subscriber-jwt.strategy.ts

```diff
@@ -17,13 +17,20 @@ export class JwtSubscriberStrategy extends PassportStrategy(Strategy, 'subscribe
     const subscriber = await this.authService.validateSubscriber(payload);
 
     if (!subscriber) {
-      throw new UnauthorizedException();
+      throw new UnauthorizedException('Subscriber validation failed.');
     }
 
     if (payload.aud !== 'widget_user') {
-      throw new UnauthorizedException();
+      throw new UnauthorizedException('Invalid token audience: Expected "widget_user".');
```

Comment:
Since this one is an internal one, I would leave it to be a little more ambiguous.